datafusion/core/src/physical_plan/aggregates/mod.rs

```diff
@@ -811,22 +811,22 @@ mod tests {
             common::collect(partial_aggregate.execute(0, task_ctx.clone())?).await?;
 
         let expected = vec![
-            "+---+---+-----------------+",
-            "| a | b | COUNT(1)[count] |",
-            "+---+---+-----------------+",
-            "|   | 1 | 2               |",
-            "|   | 2 | 2               |",
-            "|   | 3 | 2               |",
-            "|   | 4 | 2               |",
-            "| 2 |   | 2               |",
-            "| 2 | 1 | 2               |",
-            "| 3 |   | 3               |",
-            "| 3 | 2 | 2               |",
-            "| 3 | 3 | 1               |",
-            "| 4 |   | 3               |",
-            "| 4 | 3 | 1               |",
-            "| 4 | 4 | 2               |",
-            "+---+---+-----------------+",
+            "+---+-----+-----------------+",
+            "| a | b   | COUNT(1)[count] |",
+            "+---+-----+-----------------+",
+            "|   | 1.0 | 2               |",
```

Comment:
these changes are going to create some serious downstream churn I suspect. I wonder if we should wait until https://github.com/apache/arrow-rs/pull/3717 is released (so downstream crates can choose to have the old behavior) 🤔

Reply:
@alamb is the issue related to  column name `[count]`?

Reply:
Coming back to this, even with https://github.com/apache/arrow-rs/pull/3717 floats will be formatted with a decimal point, there isn't an option currently exposed to configure this...